python_modules/dagster/dagster_tests/api_tests/test_api_list_repositories.py

```diff
@@ -1,14 +1,18 @@
+import asyncio
 import sys
 
 import pytest
-from dagster._api.list_repositories import sync_list_repositories_ephemeral_grpc
+from dagster._api.list_repositories import (
+    gen_list_repositories_ephemeral_grpc,
+    sync_list_repositories_ephemeral_grpc,
+)
 from dagster._core.code_pointer import FileCodePointer, ModuleCodePointer, PackageCodePointer
 from dagster._core.errors import DagsterUserCodeProcessError
 from dagster._grpc.types import LoadableRepositorySymbol
 from dagster._utils import file_relative_path
 
 
-def test_sync_list_python_file_grpc():
+def test_list_repositories_python_file_grpc():
```

Comment:
we can add `"pytest-asyncio",` as a dependency in `[test]` and then we can have `async def` tests by adding `@pytest.mark.asyncio` 

example here https://github.com/dagster-io/internal/pull/11047/files